pkg/optimizer/optimizer.go

```diff
@@ -45,18 +46,20 @@ func NewOptimizer(env *datapath.Environment, problemData *datapath.DataInfo, sol
 }
 
 func (opt *Optimizer) getSolution(pathLength int) (string, error) {
+	opt.log.Debug().Msgf("finding solution of length %d", pathLength)
```

Comment:
Why not use the standardized fybrik logging?  Same comment for other logging calls in this file.

Reply:
It is standardized. "log" is a member of Optimizer which comes from "github.com/rs/zerolog"
This printout will look like
`{"level":"debug","Controller":"FybrikApplication","FybrikApplication":"default/test-app-prod","app.fybrik.io/app-uuid":"b848eb60-8d2a-4d58-89b3-9d3e8197978b","time":"2022-05-30T06:57:27Z","caller":"/data/ws/the-mesh-for-data/pkg/optimizer/optimizer.go:49","message":"finding solution of length 1"}`

Reply:
OK